### ISSUE TITLE
Sequence: send message to itself or polling

### ISSUE THREAD

**Issue by lixueyi83** [ArthurSonzogni/Diagon#63]
Often times, I need to send a message to itself or polling and in which case I need to do `A -> A: polling`, something like this:
```ascii
┌─┐
│A │
└┬┘
   │
   │Polling
   │-----+
   │        |
   │<---+
   │
┌┴┐
│A │
└─┘
```

However, it seems it does not work here: 
```bash
$ diagon Sequence -- "A -> A: polling"
Something went wrong!
free(): invalid next size (normal)
Aborted
```

**Comment by ArthurSonzogni** [comment]
Yes. This is probably the top 1 feature I would personally need.

**Comment by ArthurSonzogni** [comment]
I wonder if we shouldn't also support ordering the start and the end.

Proposed syntax:
```
1,2) A -> A: Task 1
3,4) A -> A: Task 2

A: 1 < 3 < 2 < 4
```

```
┌─┐
│A│
└┬┘
 │
 │ Task 1
 │---------+
 │         |
 │ Task 2  |
 │------------+
 │         |  |
 │         |  |
 │<--------+  |
 │            |
 │<-----------+
 │
┌┴┐
│A│
└─┘
```

**Comment by lixueyi83** [comment]
That would be highly appreciated, and by the way, I just started using this fancy tool and it's really so fancy! I like everything to be simple,  easy and does the job perfectly! Thank you very much!

**Comment by bjornstromberg** [comment]
just found this tool, it sure does seem to do most of the things i would need of a Sequence Diagram tool,
from a quick look this feature and a way to make a visual difference between async and sync calls is whats missing, 
but i'll create another issue for that one.

keep up the great work @ArthurSonzogni
